cg/server/api.py

```diff
@@ -356,6 +366,23 @@ def parse_flow_cell(flowcell_id):
     return jsonify(**flow_cell.to_dict(samples=True))
 
 
+@BLUEPRINT.route("/flowcells/<flow_cell_id>/sequencing_metrics", methods=["GET"])
+def get_sequencing_metrics(flow_cell_id: str):
+    """Return sequencing metrics for a flow cell."""
+
+    if not flow_cell_id:
+        return jsonify({"error": "Invalid or missing flow cell id"}), http.HTTPStatus.BAD_REQUEST
+
+    metrics: List[
+        SampleLaneSequencingMetrics
+    ] = db.get_sample_lane_sequencing_metrics_by_flow_cell_name(flow_cell_id)
+
+    if not metrics:
+        return jsonify({"error": "Sequencing metrics not found"}), http.HTTPStatus.NOT_FOUND
```

Comment:
```suggestion
    if not metrics:
        return jsonify({"error": f"Sample Lane Sequencing metrics for flow cell {flow_cell_id} not found."}), http.HTTPStatus.NOT_FOUND
```

Might be helpful with a more specific error message.

Reply:
https://github.com/Clinical-Genomics/cg/pull/2268/commits/914ca2c412545bd8b9e2c28ae3c74483f025bd35